pages/contact.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact</title>
+  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon">
+  <link href="/styles/style.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header-container">
+    <h1 id="page-title">Esther's Portfolio</h1>
+    <nav class="nav">
+      <ul>
+        <li><a href="index.html">Home</a> |</li>
+        <li><a href="about.html">About Me</a> |</li>
+        <li><a href="portfolio.html">Portfolio</a> |</li>
+        <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sed cras ornare arcu dui vivamus arcu felis. Maecenas accumsan lacus vel facilisis volutpat est velit. Turpis tincidunt id aliquet risus feugiat. In iaculis nunc sed augue lacus viverra vitae. Sagittis id consectetur purus ut faucibus pulvinar. Ipsum a arcu cursus vitae congue mauris. Duis ultricies lacus sed turpis tincidunt id aliquet risus feugiat. Risus at ultrices mi tempus imperdiet nulla malesuada pellentesque. Tristique nulla aliquet enim tortor. Aliquet risus feugiat in ante metus dictum at. Fames ac turpis egestas integer eget aliquet nibh. Magna etiam tempor orci eu lobortis elementum nibh tellus. Quis varius quam quisque id diam vel. Enim nec dui nunc mattis enim ut tellus elementum. Hendrerit dolor magna eget est lorem ipsum.</p>
+  </main>
+
+  <footer>
+    <div class="footer">
+      <p>© Esther Annorzie, 2022</p>
+      <a class="social-media" href="https://facebook.com/estherannorzie"><p>Facebook</p></a>
+      <a href="https://linkedin.com/in/estherannorzie"><p>LinkedIn</p></a>  
+      <a href="https://github.com/estherannorzie"><p>GitHub</p></a>
+    </div>
```

Comment:
I'm wondering if this `div` each of the page footers have is necessary. Since there's only one css rule they're targeted in, could we remove the `div` and target the `footer` element itself in the CSS?